Makefile.am

```diff
@@ -109,8 +109,10 @@ EXTRA_DIST += \
 #
 # usbguard icon
 #
-.svg.o:
-	$(LD) -r -b binary -o $@ $<
+$(top_builddir)/src/usbguard-icon.hpp: $(top_builddir)/icons/usbguard-icon.svg
+	echo -e "#ifndef ICON_HPP\n#define ICON_HPP\nnamespace notify {\nconst char *icon =" > $@
+	$(SED) 's/"/\\"/g' $^ | $(SED) 's/^/"/' | $(SED) 's/$$/"/' >> $@
+	echo -e ";\n}\n#endif" >> $@
 
```

Comment:
If the hpp file will be generated dynamically then you will need to add icons/usbguard-icon.svg to the tarball(```EXTRA_DIST```)